src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -125,6 +138,16 @@ private void proxyWmsRequest(HttpServletRequest request, HttpServletResponse res
         }
     }
 
+    private boolean isCollectionWithTimeMismatch(String layerName) {
```

Comment:
Are we java 1.8 now? 

Is it possible to write this as a filter and predicate? 

http://stackoverflow.com/questions/122105/what-is-the-best-way-to-filter-a-java-collection